package/endpoint_router/endpoint_router/src/endpoint_router_load.c

```diff
@@ -347,6 +367,33 @@ static PROCESS_FN(port_name)
   return event_port_string(parser, context, assign_name);
 }
 
+static bool is_valid_metric_name(const char *s)
+{
+  if (s == NULL) return false;
+
+  size_t s_size = strlen(s);
+  if (s_size == 0) return false;
```

Comment:
This failure is not logged. And error string suggests it was meant to be logged.